hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/embedded/EmbeddedTimelineService.java

```diff
@@ -100,6 +100,10 @@ public void startServer() throws IOException {
                   * writeConfig.getHoodieClientHeartbeatTolerableMisses());
     }
 
+    if (writeConfig.isTimelineServerBasedInstantStateEnabled()) {
+      timelineServiceConfBuilder.enableCkpMetadataRequests(true);
```

Comment:
Fix the method name.

Reply:
done. And should we rename all `ckpMetadata` in timeline server to `InstantState`? for example in `CkpMetadataHandler` there are a lot of ckpmetadata keywords.